Library/Formula/consul.rb

```diff
@@ -13,6 +13,8 @@ class Consul < Formula
     sha256 "67052157266d536598ba7eb5af31a10fd3e609ca812be6d50869d0fe80a583f9" => :mountain_lion
   end
 
+  option "with-web-ui", "Installs the consul web ui (to /usr/local/share/consul directory)"
```

Comment:
The use of `pkgshare` is fine. It's just that `HOMEBREW_PREFIX` can be anything, it's usually `/usr/local` but you can make it `/opt/brew` or `~/homebrew` or `/Library/Shiny/brew` or whatever you like really, so the path could potentially be `/opt/brew/share/consul`.

It's probably best to leave the path out, we don't really include them in options.